checkout/v3/index.md

```diff
@@ -1,48 +1,125 @@
-  url:  /checkout/v3/authenticated/
-  icon:
-    content: shopping_cart
-    outlined: true
-- title: Merchant Authenticated Consumer
-  description: Use this implementation if you have consumer data and consumer authentication on your site, but want to access Swedbank Pay Checkout for an easier purchase flow.
-  url:  /checkout/v3/mac/
-  icon:
-    content: shopping_cart
-    outlined: true
+  Before moving on we would like to give you a brief introduction to what you need
+  to consider before composing your checkout page, along with some prerequisites.
+menu_order: 300
 ---
 
-{:.heading-line}
-{% include card-list.html card_list=page.card_list col_class="col-lg-4" %}
+## Choose The Right Implementation For Your Business
 
-## Prerequisites
+Truth is, the customer journey varies a lot depending on your business
+vertical. For example, if you're selling physical goods like clothes, shoes or
+computers, you need to collect the address of the consumer for shipping
+purposes. Unlike if you were selling digital goods, where that's simply not
+necessary. But it also comes down to what you are able and wish to manage
+yourself.
 
-To start integrating Swedbank Pay Checkout, you need:
+When building your checkout page you have two main paths. Either you let us
+provide you with the entire checkout solution. This includes consumer
+identification and payment menu, where your customer themselves can choose how
+to pay. Or, you can choose to only use our payment option (also called the
+payment menu), where you are in charge of collecting and storing the customer
+data.
 
-*   An [HTTPS][https] enabled web server.
-*   An agreement that includes Swedbank Pay Checkout.
-*   Obtained credentials (Merchant Access Token) from Swedbank Pay through
-    Swedbank Pay Admin.
+Regardless if you choose our checkout or payments only option, you will always
+be able to decide which payment options are available for the consumer. Hence
+you can show all available payment options for that specific market, or just one
+or two if that makes more sense for your business.
 
-[https]: /introduction#connection-and-protocol
+**Consumer Info:**  The consumer's personal data e.g. name, address, phone
+number etc.
+
+**Authendication:** The process for verifying the consumer's identity via
+Strong Consumer Authentication (e.g. BankID).
+
+**Delivery Info:** Information about where the goods should be delivered.
+
+**PSP:** The service of providing payment methods in the checkout or payment
+menu.
+
+## What Are You Looking For?
+
+**Full Checkout**
+**Payments Only**
+
+## Full Checkout
+
+By using the Full Checkout, we help you collect and safely store consumer data.
+We can also prefill consumer info in the checkout if they have agreed to let us
+store their info. All of our implementations support both single payment options
+and the full payment offering.
+
+**Standard**
```

Comment:
remark-lint is reacting to this line where emphasis is being used instead of a heading. 

```suggestion
### Standard
```